SdsRestApiCore/appsettings.placeholder.json

```diff
@@ -1,8 +1,9 @@
 ﻿{
   "NamespaceId": "REPLACE_WITH_NAMESPACE_ID",
+  "CommunityId": "",
```

Comment:
I would add "REPLACE_WITH_COMMUNITY_ID_OR_REMOVE" to make this consistent with the rest of the placeholders

Reply:
the code checks for 'isnullorempty' though, so I'd lean towards how it's set up now. The placeholder value of an optional  parameter will silent get skipped. if the entire entry is removed, would like like throw an exception?
`string communityId = _configuration["CommunityId"];`

Reply:
Collin I actually started with that, but realized it might not be worth the risk someone would leave it there and the code would try to find the "REPLACE_WITH_COMMUNITY_ID_OR_REMOVE" community. If the entire entry is removed, Kelsey is right that it would throw an exception here. @derekendres would like your opinion here as well, I guess you've already approved it, but are you comfortable with the empty string placeholder so that it just skips by default?
I could also always hard-code it to ignore the specific placeholder value, it just feels clunky to do it that way.

Reply:
instead of checking if the string is nullorempty, it could check if the string is a proper guid? I'm fine with it being empty, I'm just brainstorming this out there

Reply:
That's true. We don't do that in any of the other samples, but C# does have a `Guid.TryParse()` method that could be used here. I'm still not sure I'd add the placeholder back though since we aren't including one in the other samples.

Reply:
I think I'll leave it as-is instead of adding an extra guid validity check that wouldn't be present in the other language samples.